README.md

```diff
@@ -13,14 +13,22 @@ See the [getting started instructions](https://www.lf-lang.org/embedded-lab/Gett
 To run basic tests and monitor GPIO, UART, and other supported peripherals, a [Node.js](https://nodejs.org/en)-based [emulator for the Raspberry Pi Pico](https://docs.wokwi.com/parts/wokwi-pi-pico) is provided in this repo. During the `nix` shell setup for the repo, the Node modules in the `test` directory are installed. This emulator is based on [Wokwi](https://wokwi.com).
 Because it emulates the Pico board, not the Pololu robot, it will not be able to run all the programs for the robot.
 
-By default, the emulator uses **hex** binaries which are generated by both build options. Any hex files that are in need of testing must be placed in the ``/target/hex/`` directory.
+By default, the emulator uses **hex** binaries which are generated by both build options. Any hex files that are in need of testing must be placed in the ``/test`` directory.
 
-Run the following from the ``/test/`` directory. It will run an emulator instance for each hex file in the ``hex`` directory in parallel and report results as plain text files.
+Run the following from the ``/test`` directory. It will run an emulator instance for each hex file in the directory in parallel and report results as plain text files in the same directory.
```

Comment:
I just tried this and HelloPico.hex fails, but no text files are created.

Reply:
Is this still an issue? I'm seeing a txt output in the form of "HelloPico.hex_out.txt" in the test folder on failed tests.

Reply:
Yes, this seems to work now.